control-plane/control-plane-defaults/main.yaml

```diff
@@ -4,3 +4,4 @@ metal_control_plane_ingress_dns:
 metal_control_plane_stage_name: test
 metal_control_plane_namespace: metal-control-plane
 metal_control_plane_image_pull_policy: IfNotPresent
+gardener_seeds_virtual_garden_kubeconfig: "{{ lookup('k8s', api_version='v1', kind='Secret', namespace='garden', resource_name='garden-kubeconfig-for-admin').get('data', {}).get('kubeconfig') | b64decode }}"
```

Comment:
The variable prefix does not follow the prefix convention. I'd suggest to just add an own variable in every role for this kubeconfig. This produces some kind of redundancy but I think it's in general better when roles are as isolated as possible. E.g. it's unclear when this variable can be used, it depends on running a Gardener deployment role first, but how do people know?